FetchMigration/python/orchestrator.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+import logging
+import os
+import subprocess
+
+import migration_monitor
+import pre_migration
+from migration_monitor_params import MigrationMonitorParams
+from pre_migration_params import PreMigrationParams
+
+
+__DP_EXECUTABLE_SUFFIX = "/bin/data-prepper"
+__PIPELINE_OUTPUT_FILE_SUFFIX = "/pipelines/pipeline.yaml"
+
+
+def run(dp_base_path: str, dp_config_file: str, dp_endpoint: str):
+    dp_exec_path = dp_base_path + __DP_EXECUTABLE_SUFFIX
+    output_file = dp_base_path + __PIPELINE_OUTPUT_FILE_SUFFIX
+    pre_migration_params = PreMigrationParams(dp_config_file, output_file, report=True)
+    logging.info("Running pre-migration steps...\n")
+    pre_migration_result = pre_migration.run(pre_migration_params)
```

Comment:
Please come up with a name that is more descriptive.  Everything is a migration, including the "pre" steps in many cases.  Say something like "index_config_migration" or "detecting_configurations", etc.

Reply:
Changed the notion of "pre-migration" to "metadata migration" throughout